.github/workflows/release.yaml

```diff
@@ -35,7 +35,13 @@ jobs:
       fail-fast: true
       matrix:
         # binaries wanted: linux amd64, mac intel, mac M1, windows x86
-        os: [ ubuntu-latest, macos-13, macos-14, windows-latest ]
+        os: [ ubuntu-latest, macos-latest, macos-13, macos-14, windows-latest ]
```

Comment:
Unless I'm wrong, `macos-14` and `macos-15` have the same architecture, so while I'm fine dual testing those two platforms, having separate distributions makes no sense imho.

Reply:
so I guess, drop the macos-14 should be fine?